pygitguardian/client.py

```diff
@@ -538,6 +540,71 @@ def multi_content_scan(
 
         return obj
 
+    def scan_and_create_incidents(
+        self,
+        documents: List[Dict[str, str]],
+        source_uuid: UUID,
+        extra_headers: Optional[Dict[str, str]] = None,
```

Comment:
Just missing that `*` to make `extra_headers` keyword-only.

```suggestion
        *,
        extra_headers: Optional[Dict[str, str]] = None,
```